### ISSUE TITLE
Missing stack trace

### ISSUE THREAD

**Issue by vivekhumnabad** [getsentry/sentry-cocoa#4576]
### Platform

macOS

### Environment

Production

### Installed

CocoaPods

### Version

8.36.0

### Xcode Version

15.2

### Did it work on previous versions?

No

### Steps to Reproduce

No Steps to repro , we see this randomly

### Expected Result

Full crash report with all thread details

### Actual Result

Randomly we are seeing stack traces missing in crash report -  only one thread is shown and that too with limited info

https://citrix-workspace.sentry.io/issues/5882669788/?project=4507448592105472&query=is%3Aarchived&referrer=issue-stream&sort=freq&statsPeriod=14d&stream_index=4

Event id:384424173609426ea8b0168ad1bdba65

**Comment by philipphofmann** [comment]
@vivekhumnabad, please share your SDK init configuration. Maybe you have some feature enabled that could cause this.

**Comment by vivekhumnabad** [comment]
@philipphofmann  here is the init info
            options.enableAppHangTracking = false
            options.enableNetworkTracking = false
            options.enableFileIOTracing = false
            options.enableSwizzling = false
            options.enableCoreDataTracing = false
            options.environment = self.getEnvironment()
            options.appHangTimeoutInterval = Self.appHangTimeoutInterval

**Comment by philipphofmann** [comment]
@vivekhumnabad, OK, so you have most features disabled. I'm looking for information to narrow the search for the root cause. Does this happen randomly for all types of crashes, or maybe only a few specific crashes?

**Comment by vivekhumnabad** [comment]
We have seen this in couple of crashes

**Comment by philipphofmann** [comment]
> We have seen this in couple of crashes

Yes, , but random types of crashes or a couple of crashes of the same type? For example, only EXC_BAD_ACCESS, or unhandled NSExceptions, or SIGSEV, etc.

**Comment by vivekhumnabad** [comment]
Another crash with different type
https://citrix-workspace.sentry.io/issues/5911700290/?project=4507448592105472&query=is%3Aarchived&referrer=issue-stream&statsPeriod=14d&stream_index=7
So we found one with SIGABRT and one with SEGV_NOOP

**Comment by philipphofmann** [comment]
@vivekhumnabad, thanks for the update. The two events you shared are with Cocoa SDK version 8.29.1. Although I can't find a specific bug fix related to this issue, we fixed plenty of bugs since then. It could be that the problem is already fixed. Do you also see the same behavior on recent Cocoa SDK versions?

**Comment by kahest** [comment]
Closing due to inactivity - reply with updated information if this is still relevant.
